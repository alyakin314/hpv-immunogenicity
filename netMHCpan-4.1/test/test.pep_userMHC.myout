# /Users/antonalyakin/Documents/Code/tanya/netMHCpan-4.1/Darwin_x86_64/bin/netMHCpan -hlaseq B0702.fsa -p test.pep
# Sun Dec 19 09:31:24 2021
# User: antonalyakin
# PWD : /Users/antonalyakin/Documents/Code/tanya/netMHCpan-4.1/test
# Host: Darwin Antons-MBP-2.lan 19.3.0 x86_64
# -hlaseq  B0702.fsa            File with full length MHC sequences
# -p       1                    Use peptide input
# Command line parameters set to:
#	[-rdir filename]     /Users/antonalyakin/Documents/Code/tanya/netMHCpan-4.1/Darwin_x86_64 Home directory for NetMHpan
#	[-syn filename]      /Users/antonalyakin/Documents/Code/tanya/netMHCpan-4.1/Darwin_x86_64/data/synlist.bin Synaps file
#	[-v]                 0                    Verbose mode
#	[-dirty]             0                    Dirty mode, leave tmp dir+files
#	[-tdir filename]     /var/folders/qq/z3kj6fy90wnf9fhw1vwp7n480000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)
#	[-hlapseudo filename] /Users/antonalyakin/Documents/Code/tanya/netMHCpan-4.1/Darwin_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences
#	[-hlaseq filename]   B0702.fsa            File with full length MHC sequences
#	[-a line]            HLA-A02:01           MHC allele
#	[-f filename]                             File name with input
#	[-w]                 0                    w option for webface
#	[-s]                 0                    Sort output on descending affinity
#	[-p]                 1                    Use peptide input
#	[-rth float]         0.500000             Rank Threshold for high binding peptides
#	[-rlt float]         2.000000             Rank Threshold for low binding peptides
#	[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)
#	[-xls]               0                    Save output to xls file
#	[-xlsfile filename]  NetMHCpan_out.xls    Filename for xls dump
#	[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]
#	[-thrfmt filename]   /Users/antonalyakin/Documents/Code/tanya/netMHCpan-4.1/Darwin_x86_64/data/threshold/%s.thr.%s Format for threshold filenames
#	[-expfix]            0                    Exclude prefix from synlist
#	[-version filename]  /Users/antonalyakin/Documents/Code/tanya/netMHCpan-4.1/Darwin_x86_64/data/version File with version information
#	[-inptype int]       0                    Input type [0] FASTA [1] Peptide
#	[-listMHC]           0                    Print list of alleles included in netMHCpan
#	[-allname filename]  /Users/antonalyakin/Documents/Code/tanya/netMHCpan-4.1/Darwin_x86_64/data/allelenames File with print names for alleles
#	[-BA]                0                    Include Binding affinity prediction

# NetMHCpan version 4.1b

# Tmpdir made /var/folders/qq/z3kj6fy90wnf9fhw1vwp7n480000gn/T//netMHCpanYAytBL
# Use user MHC pseudo sequence YYSEYRNIYAQTDESNLYLSYDYYTWAERAYEWY USER_DEF
# Input is in PEPTIDE format

# Make EL predictions

USER_DEF : Distance to training data  0.000 (using nearest neighbor HLA-B07:02)

---------------------------------------------------------------------------------------------------------------------------
 Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL      Exp BindLevel
---------------------------------------------------------------------------------------------------------------------------
   1    USER_DEF      AAAWYLWEV AAAWYLWEV  0  0  0  0  0    AAAWYLWEV         PEPLIST 0.0008220  -99.900  0.74258
   1    USER_DEF      AAGLQDCTM AAGLQDCTM  0  0  0  0  0    AAGLQDCTM         PEPLIST 0.0022250  -99.900  0.00000
   1    USER_DEF      AARNIVRRA AARNIVRRA  0  0  0  0  0    AARNIVRRA         PEPLIST 0.0144320  -99.900  0.00000
   1    USER_DEF      AARPDDPTL AARPDDPTL  0  0  0  0  0    AARPDDPTL         PEPLIST 0.3879260  -99.900  0.14875
   1    USER_DEF      AASCGGAVF AASCGGAVF  0  0  0  0  0    AASCGGAVF         PEPLIST 0.0342610  -99.900  0.14922
   1    USER_DEF      AASKQQMLM AASKQQMLM  0  0  0  0  0    AASKQQMLM         PEPLIST 0.0317940  -99.900  0.00000
   1    USER_DEF      AASSTHRKV AASSTHRKV  0  0  0  0  0    AASSTHRKV         PEPLIST 0.0140160  -99.900  0.27167
   1    USER_DEF      AEALLADGL AEALLADGL  0  0  0  0  0    AEALLADGL         PEPLIST 0.0011400  -99.900  0.08469
   1    USER_DEF      AEESLSLEA AEESLSLEA  0  0  0  0  0    AEESLSLEA         PEPLIST 0.0003850  -99.900  0.00000
   1    USER_DEF      AEFGPWQTV AEFGPWQTV  0  0  0  0  0    AEFGPWQTV         PEPLIST 0.0095390  -99.900  0.31562
---------------------------------------------------------------------------------------------------------------------------

Protein PEPLIST. Allele USER_DEF. Number of high binders 0. Number of weak binders 0. Number of peptides 10

-----------------------------------------------------------------------------------
