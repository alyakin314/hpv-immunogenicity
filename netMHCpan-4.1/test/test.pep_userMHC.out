# /usr/cbs/bio/src/netMHCpan-4.1/4.1b/Linux_x86_64/bin/netMHCpan -hlaseq B0702.fsa -p test.pep
# Tue Jul 28 10:13:47 2020
# User: mniel
# PWD : /usr/cbs/bio/src/netMHCpan-4.1/4.1b/test
# Host: Linux padawan 3.0.101-105-desktop x86_64
# -hlaseq  B0702.fsa            File with full length MHC sequences
# -p       1                    Use peptide input
# Command line parameters set to:
#	[-rdir filename]     /usr/cbs/bio/src/netMHCpan-4.1/4.1b/Linux_x86_64 Home directory for NetMHpan
#	[-syn filename]      /usr/cbs/bio/src/netMHCpan-4.1/4.1b/Linux_x86_64/data/synlist.bin Synaps file
#	[-v]                 0                    Verbose mode
#	[-dirty]             0                    Dirty mode, leave tmp dir+files
#	[-tdir filename]     /scratch/netMHCpanXXXXXX Temporary directory (made with mkdtemp)
#	[-hlapseudo filename] /usr/cbs/bio/src/netMHCpan-4.1/4.1b/Linux_x86_64/data/MHC_pseudo.dat File with MHC pseudo sequences
#	[-hlaseq filename]   B0702.fsa            File with full length MHC sequences
#	[-a line]            HLA-A02:01           MHC allele
#	[-f filename]                             File name with input
#	[-w]                 0                    w option for webface
#	[-s]                 0                    Sort output on descending affinity
#	[-p]                 1                    Use peptide input
#	[-rth float]         0.500000             Rank Threshold for high binding peptides
#	[-rlt float]         2.000000             Rank Threshold for low binding peptides
#	[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)
#	[-xls]               0                    Save output to xls file
#	[-xlsfile filename]  NetMHCpan_out.xls    Filename for xls dump
#	[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]
#	[-thrfmt filename]   /usr/cbs/bio/src/netMHCpan-4.1/4.1b/Linux_x86_64/data/threshold/%s.thr.%s Format for threshold filenames
#	[-expfix]            0                    Exclude prefix from synlist
#	[-version filename]  /usr/cbs/bio/src/netMHCpan-4.1/4.1b/Linux_x86_64/data/version File with version information
#	[-inptype int]       0                    Input type [0] FASTA [1] Peptide
#	[-listMHC]           0                    Print list of alleles included in netMHCpan
#	[-allname filename]  /usr/cbs/bio/src/netMHCpan-4.1/4.1b/Linux_x86_64/data/allelenames File with print names for alleles
#	[-BA]                0                    Include Binding affinity prediction

# NetMHCpan version 4.1b

# Tmpdir made /scratch/netMHCpanY0SYsQ
# Use user MHC pseudo sequence YYSEYRNIYAQTDESNLYLSYDYYTWAERAYEWY USER_DEF
# Input is in PEPTIDE format

# Make EL predictions

USER_DEF : Distance to training data  0.000 (using nearest neighbor HLA-B07:02)

---------------------------------------------------------------------------------------------------------------------------
 Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL      Exp BindLevel
---------------------------------------------------------------------------------------------------------------------------
   1    USER_DEF      AAAWYLWEV AAAWYLWEV  0  0  0  0  0    AAAWYLWEV         PEPLIST 0.0008220  -99.900  0.74258
   1    USER_DEF      AAGLQDCTM AAGLQDCTM  0  0  0  0  0    AAGLQDCTM         PEPLIST 0.0022250  -99.900  0.00000
   1    USER_DEF      AARNIVRRA AARNIVRRA  0  0  0  0  0    AARNIVRRA         PEPLIST 0.0144320  -99.900  0.00000
   1    USER_DEF      AARPDDPTL AARPDDPTL  0  0  0  0  0    AARPDDPTL         PEPLIST 0.3879260  -99.900  0.14875
   1    USER_DEF      AASCGGAVF AASCGGAVF  0  0  0  0  0    AASCGGAVF         PEPLIST 0.0342610  -99.900  0.14922
   1    USER_DEF      AASKQQMLM AASKQQMLM  0  0  0  0  0    AASKQQMLM         PEPLIST 0.0317940  -99.900  0.00000
   1    USER_DEF      AASSTHRKV AASSTHRKV  0  0  0  0  0    AASSTHRKV         PEPLIST 0.0140160  -99.900  0.27167
   1    USER_DEF      AEALLADGL AEALLADGL  0  0  0  0  0    AEALLADGL         PEPLIST 0.0011400  -99.900  0.08469
   1    USER_DEF      AEESLSLEA AEESLSLEA  0  0  0  0  0    AEESLSLEA         PEPLIST 0.0003850  -99.900  0.00000
   1    USER_DEF      AEFGPWQTV AEFGPWQTV  0  0  0  0  0    AEFGPWQTV         PEPLIST 0.0095390  -99.900  0.31562
---------------------------------------------------------------------------------------------------------------------------

Protein PEPLIST. Allele USER_DEF. Number of high binders 0. Number of weak binders 0. Number of peptides 10

-----------------------------------------------------------------------------------
